presto-main/src/main/java/com/facebook/presto/execution/scheduler/SqlQueryScheduler.java

```diff
@@ -216,6 +226,43 @@ private SqlQueryScheduler(
 
         this.stages = stages.stream()
                 .collect(toImmutableMap(SqlStageExecution::getStageId, identity()));
+        this.stageByFragmentIdMap = stages.stream()
+                .collect(toImmutableMap(stage -> stage.getFragment().getId(), identity()));
+
+        Set<PlanFragmentId> rootIds = new HashSet<>();
+        rootIds.add(plan.getFragment().getId());
+        // dependencies are roots
+        plan.geAllStages()
```

Comment:
Is `ge` short for something or is this a typo?

Reply:
That's a typo. Let me fix it

Reply:
This code has been removed